### DISCUSSION TITLE
Add a Tutorial Pane, like the one in RStudio

### DISCUSSION THREAD

**Discussion by davidkane9** [posit-dev/positron#5660] [Feedback]
The RStudio Tutorial Pane works well. It is especially useful for beginner students, those who are technical enough to press the "Start Tutorial" but not technical enough to run a command at the R Console. It would be nice if Positron had a Tutorial Pane.

**Comment by juliasilge** [comment]
Thanks for this request! 🙌 I have opened https://github.com/posit-dev/positron/issues/5668 to track and prioritize this feature.
